src/f64x2_.rs

```diff
@@ -438,7 +438,7 @@ impl f64x2 {
   #[must_use]
   pub fn fast_max(self, rhs: Self) -> Self {
     pick! {
-      if #[cfg(target_feature="sse")] {
+      if #[cfg(target_feature="sse2")] {
```

Comment:
This wouldn't compile otherwise with just sse set the wrong fields are declared

Reply:
Ah, yeah there's probably not much testing in the crate for `sse` but *not* `sse2`. I think that in the long term `f64x2` should have its definition adjusted so that it uses an `m128d` field when just `sse` is set, but that can wait for later since it's kinda unrelated to this PR.